sklearn/tests/test_calibration.py

```diff
@@ -341,3 +342,24 @@ def decision_function(self, X):
     calibrated_clf = CalibratedClassifierCV(MockTensorClassifier())
     # we should be able to fit this classifier with no error
     calibrated_clf.fit(X, y)
+
+
+def test_calibration_pipeline():
+    """Test that calibration works in pre-fit pipeline with transformer,
```

Comment:
Use comments instead of docstring. It will not have a nice interaction with pytest when showing the test names

Reply:
Actually this was true with nosetests (that decided that to print the test name when running the test and there was an additional step to figure out the test function) and is fine with pytest but overall not a big deal :shrug: ...

There maybe another thing I am missing of course.

Just for completeness:
```py
# test_docstring.py
def test_bla():
    "docstring for test bla"
    assert 1 + 1 == 2
```

For pytest the name of the test printed on screen is still test_bla
```
❯ pytest test_docstring.py -v
========================================= test session starts ==========================================
platform linux -- Python 3.8.3, pytest-5.4.3, py-1.8.2, pluggy-0.13.1 -- /home/lesteve/miniconda3/envs/del/bin/python
cachedir: .pytest_cache
rootdir: /tmp
collected 1 item                                                                                       

test_docstring.py::test_bla PASSED                                                               [100%]

========================================== 1 passed in 0.00s ===========================================
```

For nosetests the test printed on screen uses the docstring
```
❯ nosetests test_docstring.py -v
docstring for test bla ... ok

----------------------------------------------------------------------
Ran 1 test in 0.000s

OK
```

Reply:
OK I am getting old ;)

Reply:
@lesteve you are right it was my old memories with testing and it was with `nose`